web/src/components/repo/RepoCard.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <router-link
+    v-if="repo"
+    :to="{ name: 'repo', params: { repoId: repo.id } }"
+    class="flex flex-col border rounded-md bg-wp-background-100 overflow-hidden p-4 border-wp-background-400 dark:bg-wp-background-200 cursor-pointer hover:shadow-md hover:bg-wp-background-300 dark:hover:bg-wp-background-300"
+  >
+    <div class="grid grid-cols-[auto,1fr] gap-y-1 items-center">
+      <div class="text-wp-text-100 text-lg">{{ `${repo.owner} / ${repo.name}` }}</div>
+      <div class="ml-auto">
+        <Badge v-if="repo.visibility === RepoVisibility.Public" :label="$t('repo.visibility.public.public')" />
+      </div>
+
+      <div class="col-span-2 text-wp-text-100">
+        <div v-if="lastPipeline" class="flex gap-x-1 items-center">
+          <PipelineStatusIcon v-if="lastPipeline" :status="lastPipeline.status" />
+          <span class="whitespace-nowrap overflow-hidden overflow-ellipsis">{{ shortMessage }}</span>
+        </div>
+
+        <div v-else class="flex gap-x-2">
+          <span>{{ $t('repo.pipeline.no_pipelines') }}</span>
+        </div>
+      </div>
+
+      <div class="col-span-2 text-wp-text-100 text-sm min-h-5 mt-3">
```

Comment:
```suggestion
      <div class="col-span-2 text-wp-text-100 text-sm min-h-4 mt-4">
```

Spacings / heights etc should try to be a multiple of 2.

Reply:
I don't agree in general but fine for me.